packages/template-chakra-storefront/config/default.json

```diff
@@ -14,91 +14,121 @@
             "siteId": "RefArchGlobal"
         }
     },
+    "mobify": {
+        "app": {
+        "extensions": []
+        },
+        "ssrEnabled": true,
+        "ssrOnly": [
+        "ssr.js",
+        "ssr.js.map",
+        "node_modules/**/*.*"
+        ],
+        "ssrShared": [
+        "static/favicon.ico",
+        "static/robots.txt",
+        "**/*.js",
+        "**/*.js.map",
+        "**/*.json"
+        ],
+        "ssrParameters": {
+        "ssrFunctionNodeVersion": "22.x",
+        "proxyConfigs": [
+            {
+            "host": "kv7kzm78.api.commercecloud.salesforce.com",
+            "path": "api"
+            },
+            {
+            "host": "zzrf-001.dx.commercecloud.salesforce.com",
+            "path": "ocapi"
+            }
+        ]
+        }
+    },
     "defaultSite": "RefArchGlobal",
     "defaultAppLocale": "en-US",
     "defaultSiteTitle": "Retail React App",
-    "defaultDnt": true,
-    "einsteinAPI": {
-        "host": "https://api.cquotient.com",
-        "einsteinId": "1ea06c6e-c936-4324-bcf0-fada93f83bb1",
-        "siteId": "aaij-MobileFirst",
-        "isProduction": false
-    },
-    "dataCloudAPI": {
-        "appSourceId": "fb81edab-24c6-4b40-8684-b67334dfdf32",
-        "tenantId": "mmyw8zrxhfsg09lfmzrd1zjqmg"
-    },
-    "login": {
-        "passwordless": {
-            "enabled": false,
-            "callbackURI": "/passwordless-login-callback",
-            "landingPath": "/passwordless-login-landing"
-        },
-        "social": {
-            "enabled": false,
-            "idps": ["google", "apple"],
-            "redirectURI": "/social-callback"
-        },
-        "resetPassword": {
-            "callbackURI": "/reset-password-callback",
-            "landingPath": "/reset-password-landing"
-        }
-    },
-    "maxCacheAge": 900,
     "pages": {
-        "Account": {
+        "account": {
             "path": "/account",
             "orderSearchParam": {"limit": 10, "offset": 0, "sort": "best-matches", "refine": []}
         },
-        "Cart": {
+        "cart": {
```

Comment:
Where do we use this config? Maybe we don't need it anymore 🤔

Reply:
Not all of them are used. But some definitely are. Make sure not to delete the whole pages object

Reply:
I checked the route.jsx. None of the pages here are used there. The only route that is used is related to pwless/resetpw login section below